### PR TITLE
Misc site updates

### PR DESCRIPTION
**Pull request by truemiller**
resources updates:
- added Olas Dune dashboard
- added staking Dune dashboard

podcast update: 
- added recent podcast

friends of olas update:
- added nevermined, naptha, origin trail, kleros, safe, flock.io & builders

builders update:
- added builders subsection to ecosystem
- added 8baller, nevermined, naptha, valory, gnosis

build page:
- added section id "rewards" to build rewards
- fixed typo copied from google doc
- limited text content width to 800px

### CODE REVIEW

File: public/images/friends/kleros.png
Comment (truemiller) [inline public/images/friends/kleros.png:1]: found this png via kleros' brand assets, as wasn't added to ticket, please confirm this is ok

File: components/HomepageSection/Services.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import services from "@/data/services.json";
 import serviceCategories from "@/data/serviceCategories.json";
 import core from "@/data/core.json";
 import chains from "@/data/chains.json";
-import Badge from "../Badge";
```
Comment (truemiller) [inline components/HomepageSection/Services.jsx:7]: removed as was unused

File: data/friends.json [hunk 2]
```diff
@@ -58,5 +58,53 @@
...
+  {
+    "id": "2d21690f-0500-4bcc-9caa-40b0ff7e8c72",
+    "name": "Origin Trail",
+    "url": "https://www.origintrail.io/",
+    "imageFilename": "origin-trail.png"
+  },
+  {
+    "id": "6853c5f6-eb3f-4671-b9b4-94d896cf1688",
+    "name": "Omen",
+    "url": "https://omen.eth.link/",
```
[the remark this answers is not in the document]
Comment (oaksprout) [reply data/friends.json:83]: You can use this one: https://aiomen.eth.limo/#/liquidity

[the remark this answers is not in the document]
Comment (oaksprout) [reply data/friends.json:83]: It's optimised for our agents

File: public/images/friends/kleros.png
Comment (oaksprout) [reply public/images/friends/kleros.png:1]: My bad, missed that one

Comment (oaksprout) [reply public/images/friends/kleros.png:1]: Seems like the most up to date to me.

File: styles/globals.css [hunk 3]
```diff
@@ -65,3 +65,18 @@
     --ring: 212.7 26.8% 83.9%;
   }
 }
+
+.image-container {
```
Comment (oaksprout) [inline styles/globals.css:69]: Any reason to do this with css rather than Tailwind?

File: components/HomepageSection/Services.jsx [hunk 4]
```diff
@@ -195,6 +197,36 @@ const ChainsSubsection = () => {
...
+    <SectionWrapper id="builders">
+      <div className="max-w-screen-xl mb-12 mx-auto">
+        <div className="grid grid-cols-1 gap-6 lg:grid-cols-1">
+          <div>
+            <h2 className="text-3xl font-bold mb-2">
+              Builders
+            </h2>
+
+            <p className="mt-4 text-xl font-light md:text-3xl lg:text-xl text-gray-600 max-w-[700px]">
+              Olas has a growing ecosystem of talented developer organizations, pushing the autonomous edge every day, build autonomous agents and services.
```
Comment (oaksprout) [inline components/HomepageSection/Services.jsx:211]:
```suggestion
              Olas has a growing ecosystem of talented developer organizations, pushing the autonomous edge every day, building autonomous agents and services.
```

Comment (oaksprout) [reply components/HomepageSection/Services.jsx:211]: Probs originally my typo!

File: components/HomepageSection/Services.jsx [hunk 5]
```diff
@@ -23,13 +23,15 @@ const Item = ({ service, category }) => {
...
         customClasses="rounded-t-xl border-t-0 border-b"
       >
-        <Image
-          src={`/images/${category}/${service.iconFilename}`}
-          alt={service.name}
-          width={300}
-          height={300}
-          className="mx-auto p-2"
-        />
+        <div style={{ width: "100%", height: 200, display: "flex" }}>
```
Comment (oaksprout) [inline components/HomepageSection/Services.jsx]: For the sake of consistency, I'd suggest doing this with Tailwind utility classes.

**Review by oaksprout — APPROVED** [review]

**Review by DavidMinarsch — DISMISSED** [review]

**Comment by oaksprout** [comment]
image

Could do with the logos being more consistently sized, and higher-res. Adriano could help you source these.

image

This could have already been here, but I think we should show three cards here rather than two.

**Review by oaksprout — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**